cpp/src/parquet/types.h

```diff
@@ -811,4 +811,10 @@ PARQUET_EXPORT SortOrder::type GetSortOrder(ConvertedType::type converted,
 PARQUET_EXPORT SortOrder::type GetSortOrder(
     const std::shared_ptr<const LogicalType>& logical_type, Type::type primitive);
 
+/// PLAIN_DICTIONARY is deprecated but used to be used as a dictionary index
+/// encoding.
```

Comment:
Please make sure the docstring explains what the function does.
If you want to explain implementation details, regular comments (`//`) should be used.

Reply:
Ah I didn't know this, changed now